packages/terra-clinical-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -62,6 +62,18 @@ const propTypes = {
    * Function that will be called with a ref to the Cell's selectable element. Parameters: `selectableRefCallback(selectableRef)`
    */
   selectableRefCallback: PropTypes.func,
+  /**
+   * Boolean indicating whether the cell should be highlighted with background styling.
+   */
+  isColumnHighlighted: PropTypes.string,
```

Comment:
Should the type be `PropTypes.bool` instead of `PropTypes.string`?

Reply:
Updated [here](https://github.com/cerner/terra-clinical/pull/796/commits/58bec54cb5437d28b3cc5f69cca242e6544f6f77#diff-c55aff50e0c8be05826ebe2785a70855520b3e32b192ff4fd1b95e9aa8b0a9c4R68)